content-src/components/Snippet/Snippet.js

```diff
@@ -0,0 +1,37 @@
+const React = require("react");
+const classNames = require("classnames");
+
+const Snippet = React.createClass({
+  onClose(e) {
+    e.preventDefault();
+    this.props.setVisibility(false);
+  },
+  render() {
+    const {visible, description, title, image} = this.props;
+    return (<div className={classNames("snippet", {"hidden": !visible})}>
```

Comment:
Hm, now that I look at it, we kind of need this to be the case to support the CSS transition. This gets a bit messy it we're handling those two cases (hidden v.s. never shown ever), let me think about it